app/src/main/java/com/teamagam/gimelgimel/app/view/ImageFullscreenActivity.java

```diff
@@ -0,0 +1,70 @@
+package com.teamagam.gimelgimel.app.view;
+
+import android.net.Uri;
+import android.os.Bundle;
+import android.view.View;
+
+import com.teamagam.gimelgimel.R;
+import com.teamagam.gimelgimel.app.GGApplication;
+import com.teamagam.gimelgimel.app.view.drawable.CircleProgressBarDrawable;
+import com.teamagam.gimelgimel.app.view.drawable.ZoomableDraweeView;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+/**
+ * A Image full-screen activity that shows and hides the system UI (i.e.
+ * status bar and navigation/system bar) with user interaction.
+ */
+public class ImageFullscreenActivity extends BaseActivity<GGApplication> {
+
+    @BindView(R.id.image_fullscreen_view)
+    ZoomableDraweeView mDraweeView;
+
+    private static final String IMAGE_URI_KEY = "IMAGE_URI";
+
+    private Uri mImageUri;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        ButterKnife.bind(this);
+
+        mImageUri = getIntent().getData();
+
+        // Note that some of these constants are new as of API 16 (Jelly Bean)
+        // and API 19 (KitKat). It is safe to use them, as they are inlined
+        // at compile-time and do nothing on earlier devices.
+        mDraweeView.setSystemUiVisibility(View.SYSTEM_UI_FLAG_LOW_PROFILE
```

Comment:
Could be extracted to method.
I like to crate "init()" method, and categorize every type of initialization. !resolve